Packs/XMCO/Classifiers/classifier-mapper-incoming-XMCO.json

```diff
@@ -0,0 +1,31 @@
+{
+    "id": "XMCO - Serenety Mapper",
+    "name": "XMCO - Serenety Mapper",
+    "type": "mapping-incoming",
+    "description": "Maps incoming XMCO alert fields.",
+    "fromVersion": "6.10.0",
+    "mapping": {
+        "XMCO Serenety Alert": {
+            "dontMapEventToLabels": false,
+            "internalMapping": {
+                "Serenety category": {
```

Comment:
Please use the OOTB field `Category`. No need to create dedicated fields that are already existing. Hence, you can remove the added incident field `XMCO Serenety category` from the PR.

Reply:
Thank you for the tips